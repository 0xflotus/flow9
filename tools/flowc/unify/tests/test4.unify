import xml;
import behaviour;

DynamicBehaviour<union XmlEvent> c= DynamicBehaviour<α58>;
α25 c= α58;
union XmlTagEvent c= α25;
α3625 c= .(.tag : α52);

finalize
