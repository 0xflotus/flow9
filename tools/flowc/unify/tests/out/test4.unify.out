α25={XmlTagEvent<> .. *}
α58={XmlEvent<> .. *}
α3625=.(.tag : α52)

Final:
α25=XmlTagEvent
α52=?
α58=XmlEvent
α3625=XmlElementEnd
